app/src/main/java/com/ichi2/apisample/ui/AudioCaptureService.java

```diff
@@ -436,14 +465,14 @@ private void stopAudioCapture() {
             }
             newFilenames.add(uri.toString());
             MainActivity.storeFilenames(this, newFilenames.toArray(new String[0]));
+            uiDbEditor.putBoolean(MainActivity.REF_DB_AFTER_CAPTURING, true);
 
             long duration = System.currentTimeMillis() - recordingStartedAt;
-            String durationStr = String.format(Locale.US, "%.2f", (double) duration / 1000d);
-            Recording recording = new Recording(uri, durationStr);
+            Recording recording = new Recording(uri, duration);
             recordings.add(recording);
             textBottom.setText(getString(R.string.recorded_files, recordings.size()));
             textLatest.setVisibility(View.VISIBLE);
-            textLatest.setText(getString(R.string.latest_file, recording.getDurationStr()));
+            textLatest.setText(getString(R.string.latest_file, recording.getDuration() / 1000d));
```

Comment:
It would be better to move this logic into special method probably instead of "magic" dividing in the code

Reply:
Good point. Fixed